org.lflang/src/lib/ts/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Would it make sense to put this file in the `reactor-ts` repo?

Reply:
Oh, that had not occurred to me before. If you would like to reuse it to enforce style in `reactor-ts` as well as in the user's code, then I can just move it there and roll that secondary objective into this PR.

Reply:
What does this configuration do? And does it in any way conflict with `prettier`?

Reply:
This configuration is for validating generated TypeScript. Since the generated code uses `reactor-ts`, it might make sense for us to want `reactor-ts` to comply with the same style rules as those used in the generated code. I think it is fairly similar to the `eslintrc` file that appeared in [that article you posted](https://blog.logrocket.com/linting-typescript-using-eslint-and-prettier/), so it seems like it should be compatible with Prettier.

Reply:
Perhaps a better question is how it meshes with `babel`, which we use to compile `reactor-ts`. I believe that there is an overlap there as I recall that `babel` also does a linting step prior to attempting a full compilation, but I'm not sure...

Reply:
There is overlap. Babel has to parse in order to compile, and so it reports syntax errors. However, of course there are reasons why ESLint is popular and has not been superseded by `tsc` or Babel, and I am pushing a fix that will allow the user to provide configration files for ESLint as well. I see no reason not to leave this feature in.